recipes/gembs/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "conda_v2" %}
+{% set sha256 = "1ead3f20a36a40c0737ec58c2e783b9c1cb6a9de0317c0009964d1b164c598a0" %}
+{% set bcftoolsversion = "1.8" %}
+{% set bcftoolssha256 = "4acbfd691f137742e0be63d09f516434f0faf617a5c60f466140e0677915fced"%}
+
+package:
+  name: gembs
+  version: {{ version }}
+
+source:
+  - url: https://github.com/karl616/gemBS/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+  - url: https://github.com/samtools/bcftools/releases/download/{{ bcftoolsversion }}/bcftools-{{ bcftoolsversion }}.tar.bz2
```

Comment:
is the bcftools source files being used ? its not obvious from the build.sh.

Reply:
Yes, there is a custom compiled bcftools version installed to the python library location. It makes use of the plugin system of bcftools.
The installation script downloads bcftools, but I found it better to refer to it like this.

Reply:
but is the script going to use this one if its there ? 
because if its going to download another one either way I think its best not to put this here.

Reply:
The script will prioritize the instance installed to the library folder, with a fall-back to the main-path.

The download process is part of a Makefile. If the folder is already populated by conda, the download step during installation will be skipped. So yes, the file referred here is the one being used.